fblldbobjcruntimehelpers.py

```diff
@@ -47,7 +47,7 @@ def functionPreambleExpressionForSelf():
   expressionForSelf = None
   if arch == 'i386':
     expressionForSelf = '*(id*)($esp+4)'
-  elif arch == 'x86_64':
+  elif arch == 'x86_64' or 'x86_64h':
```

Comment:
Instead of checking for variations at these call sites, would you mind doing a normalization inside `currentArch()`. In other words, make `currentArch()` return `'x86_64'` instead of `'x86_64h'`. Thoughts?